source/slang/hlsl.meta.slang.h

```diff
@@ -24,19 +24,28 @@ SLANG_SPLICE(kIROp_HLSLByteAddressBufferType
 SLANG_RAW(")\n")
 SLANG_RAW("struct ByteAddressBuffer\n")
 SLANG_RAW("{\n")
+SLANG_RAW("    __target_intrinsic(glsl, \"$1 = $0.length()\")\n")
 SLANG_RAW("    void GetDimensions(\n")
 SLANG_RAW("        out uint dim);\n")
 SLANG_RAW("\n")
+SLANG_RAW("    __target_intrinsic(glsl, \"$0[$1]\")\n")
 SLANG_RAW("    uint Load(int location);\n")
+SLANG_RAW("\n")
 SLANG_RAW("    uint Load(int location, out uint status);\n")
```

Comment:
The versions with status out are not supported on GLSL?

Reply:
The status return is part of the "sparse texture" feature, as I understand it. Going through and adding support for those variants would be a separate PR, since we'd want to be systematic and map everything we can over.